### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by lokiCML** [AtomLinter/linter-pylint#35]
atom -v 0.196.0
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (C:\Users\lokiCML\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:790:9)
Package.hasActivationCommands (C:\Users\lokiCML\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:715:20)
<unknown> (C:\Users\lokiCML\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:169:24)
Package.measure (C:\Users\lokiCML\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:147:15)
```

**Comment by karimjosephjr** [comment]
Also in atom -v 0.198.0, linter-pylint 0.2.1

**Comment by bilderbuchi** [comment]
Should be closed as duplicate of #34.

**Comment by karimjosephjr** [comment]
I commented on this one and not #34, #36, #37, #38 given the additional information of what versions of atom and/or pylint were impacted. This could be more helpful for developers.

**Comment by karimjosephjr** [comment]
Still in atom -v 0.200.0 and linter-pylint -v 0.2.1. [Deprecated APIs will be removed June 1st](http://blog.atom.io/2015/05/01/removing-deprecated-apis.html).

**Comment by kevinsawicki** [comment]
Fixed by https://github.com/AtomLinter/linter-pylint/pull/40 and published in `0.2.2`
